DOCS/interface-changes.rst

```diff
@@ -27,6 +27,7 @@ Interface changes
 ::
 
  --- mpv 0.38.0 ---
+    - add `--begin-vo-dragging` command
```

Comment:
```suggestion
    - add `begin-vo-dragging` command
```

Reply:
Sorry to bikeshed this one, but how come? Options have almost always been prefixed with `--` in this file.